core/src/test/scala/org/apache/spark/rdd/RDDSuite.scala

```diff
@@ -29,11 +29,12 @@ import com.esotericsoftware.kryo.KryoException
 import org.apache.hadoop.io.{LongWritable, Text}
 import org.apache.hadoop.mapred.{FileSplit, TextInputFormat}
 import org.scalatest.concurrent.Eventually
-
```

Comment:
It looks like the import grouping is broken here, so Scalastyle is going to fail. Can you fix this to be consistent with the style described at https://github.com/databricks/scala-style-guide#imports (blank line separating the third-party deps from the org.apache.spark deps, removal of the blank line within the spark deps section)?